specification/authorization/resource-manager/readme.md

```diff
@@ -237,3 +237,16 @@ java:
 regenerate-manager: true
 generate-interface: true
 ```
+
+### Tag: profile-2019-03-01-hybrid
+
+These settings apply only when `--tag=profile-2019-03-01-hybrid` is specified on the command line.
+Creating this tag to pick proper resources from the hybrid profile.
+
+``` yaml $(tag) == 'profile-2019-03-01-hybrid'
```

Comment:
@amarzavery I think we should stick to `profile-hybrid-2019-03-01`

Reply:
done.